src/print.jl

```diff
@@ -638,3 +638,9 @@ end
 function Base.show(io::IO, evaluator::NLPEvaluator)
     Base.show(io, "A JuMP.NLPEvaluator")
 end
+
+#------------------------------------------------------------------------
+## Deprecated functions to export a model
+#------------------------------------------------------------------------
+writeLP(_...) = error("writeLP is no more provided by JuMP; instead, you can check https://github.com/odow/MathOptFormat.jl")
+writeMPS(_...) = error("writeMPS is no more provided by JuMP; instead, you can check https://github.com/odow/MathOptFormat.jl")
```

Comment:
These should probably go after these: 
https://github.com/JuliaOpt/JuMP.jl/blob/c01802926c34aa30e5d272ac1063f1595084aa4a/src/JuMP.jl#L29-L37
No need for the comments.

Also, please make the functions `writeLP(args...; kwargs...)` so that it catches any usage.

As an English note, `is no more provided` isn't great. How about something like
```julia
function writeLP(args...; kargs...)
    error("writeLP has been removed from JuMP. Use `MathOptFormat.jl` instead.")
end
```